spec/features/user_spec.rb

```diff
@@ -0,0 +1,110 @@
+require 'rails_helper'
+
+RSpec.describe 'User', type: :feature do
```

Comment:
- [ ] Most of your users tests are failing also.

  ![image](https://github.com/agneta1991/social-media-app/assets/36542986/cb90d4cf-401e-46f5-94fc-7f3206cf4b36)

- [ ] **[OPTIONAL]** I think it would be better to split these show and index page tests into different files for better organization.